lib/annotate.rb

```diff
@@ -76,6 +76,10 @@ def self.setup_options(options = {})
       options[:model_dir] = ['app/models']
     end
 
+    if(options[:root_dir].empty?)
+      options[:root_dir] = ['/']
```

Comment:
Is this supposed to be Rails.root? A slash doesn't seem to a correct default value from the caller/user perspective.